driver/csiplugin/controllerserver.go

```diff
@@ -479,91 +473,52 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		scaleVol.ClusterId = PCid
 	}
 
-	glog.Infof("Scale vol create params : %v\n", scaleVol)
+	glog.Infof("volume:[%v] -  spectrum scale volume create params : %v\n", scaleVol.VolName, scaleVol)
 
 	volReqInProcess, err := cs.IfSameVolReqInProcess(scaleVol)
 	if err != nil {
 		return nil, err
 	}
 
 	if volReqInProcess {
-		return nil, status.Error(codes.Aborted, fmt.Sprintf("Volume creation already in process : %v", scaleVol.VolName))
+		glog.Errorf("volume:[%v] - volume creation already in process ", scaleVol.VolName)
+		return nil, status.Error(codes.Aborted, fmt.Sprintf("volume creation already in process : %v", scaleVol.VolName))
 	}
 
 	/* Update driver map with new volume. Make sure to defer delete */
 
 	cs.Driver.reqmap[scaleVol.VolName] = volSize
 	defer delete(cs.Driver.reqmap, scaleVol.VolName)
 
-	glog.Infof("reqmap After: %v", cs.Driver.reqmap)
+	glog.V(4).Infof("reqmap After: %v", cs.Driver.reqmap)
```

Comment:
This one is a development debug log, should probably be removed

Reply:
removed